addons/crm/models/crm_lead.py

```diff
@@ -1998,12 +2001,13 @@ def _pls_get_lead_pls_values(self, domain=[]):
             lead_results = self._cr.dictfetchall()
 
             # Get tags values
-            query = """SELECT crm_lead.id as lead_id, t.id as tag_id
+            query = """SELECT crm_lead.id as lead_id%s
                             FROM %s
-                            LEFT JOIN crm_tag_rel rel ON crm_lead.id = rel.lead_id
-                            LEFT JOIN crm_tag t ON rel.tag_id = t.id
+                            %s
                             WHERE %s order by crm_lead.team_id asc"""
-            query = sql.SQL(query % (from_clause, where_clause)).format(*args)
+            if use_tags:
+                args.append(sql.Identifier('tag_id'))
+            query = sql.SQL(query % (query_tag_select, from_clause, query_tag_join, where_clause)).format(*args)
             self._cr.execute(query, where_params)
             tag_results = self._cr.dictfetchall()
```

Comment:
this does not need to be done at all if use_tag = False
query_tag_select and query_tag_join are then useless.